docs/design/solutions/reroute-topology/use-case-reroute-topology.puml

```diff
@@ -0,0 +1,56 @@
+@startuml
+
+left to right direction
+
+:Floodlight: as FL <<system>>
+
+rectangle "Event topology" {
+    rectangle "Switch operations" {
+        (Create or update\nswitch) as CUS
+        (Activate switch) as AS
+        (Deactivate switch) as DS
+    }
+
+    rectangle "Isl operations" {
+        (Create or update\nISL) as CUI
+        (Set ISL status "moved") as IDF
```

Comment:
FL do not produce "ISL moved" event. Also "ISL moved" is not an ISL operation... this is mostly alias for "ISL DOWN" with some specific precondition.